### DISCUSSION TITLE
How to log in to GitHub

### DISCUSSION THREAD

**Discussion by nyancatda** [coder/code-server#2556] [Q&A] [answered]
Open external links are opened using the local computer, how do I configure them on the server?

**Comment by code-asher** [comment]
There's currently nothing that would make it open on the server. This
wouldn't work anyway, right? Since the browser is on the client and not
the server? Unless you're using VNC or something. 🤔

**Reply by nyancatda** [reply]
> There's currently nothing that would make it open on the server. This wouldn't work anyway, right? Since the browser is on the client and not the server? Unless you're using VNC or something. 🤔

Then this thing is really useless

**Reply by gryphs** [reply]
What makes you need it? You can forward browser-side traffic to server via JavaScript like what [cloudflareworkers.com](https://cloudflareworkers.com/) do.
But anyway I don't get a idea why you need that. Why don't open it on local?

**Comment by nooobcoder** [comment, marked as answer]
> Open external links are opened using the local computer, how do I configure them on the server?

Please be more _clear to your use case_. The topic of discussion is not so clear. Why do you want to open the link on the server side?
Please explain so that I can serve you with a better response.

If you want to open a link that is generated by the server then you need to use the **inbuilt proxy** of code-server
_Here is an use case_
If `http://192.168.0.118:1000` is your code-server url then wrap the generated link here just like this
`http://192.168.0.118:1000/proxy/generated-link-here/port-here-if-any`

**PS: Use HTTPS if your server uses HTTPS otherwise the proxy should report an error**

@h123hh **If this discussion was helpful, please mark the thread as answered and upvote. It would really inspire juniors like me.**

Thank You,
Wish you have a great day ahead.
